addons/web/static/src/public/database_manager.js

```diff
@@ -27,6 +27,20 @@ document.addEventListener("DOMContentLoaded", function () {
         }
     });
 
+    document.body.addEventListener("change", function (ev) {
+        if (ev.target.getAttribute('name') == "backup_format") {
+            ev.preventDefault();
+            const no_filestore_flag = document.querySelector(".no_filestore")
```

Comment:
if you gotta query one specific element, then it is an id you need, not a class, and `getElementById`